sdk/communication/Azure.Communication.PhoneNumbers/tests/Infrastructure/AuthMethod.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.PhoneNumbers.Tests
+{
+    public enum AuthMethod
```

Comment:
Maybe move this to the common test project and use it in identity, Sms and here.

Reply:
Done.